.pre-commit-config.yaml

```diff
@@ -31,3 +31,15 @@
     hooks:
     -   id: remove-tabs
         language_version: python2.7
+# TODO: Replace this with the actual repo
+# (https://github.com/detailyang/pre-commit-shell)
+# once it supports types and not just files ending in shell extensions
+-   repo: local
+    hooks:
+    -   id: shell-lint
+        name: Shell Syntax Check
+        description: Check Shell Syntax on ALL staged files with user friendly messages and colors
+        entry: shell-lint.sh
+        language: script
+        exclude: ^.deactivate.sh$
```

Comment:
(i assumed you were excluding the .activate.sh symlink rather than the .deactivate.sh file)